Assets/XML/Text/Civics.xml

```diff
@@ -180,7 +196,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIVIC_LEVY_ARMIES_STRATEGY</Tag>
-		<English>Extra free units, military units produced with food, faster construction of Castles</English>
+		<English>Lower unit support costs; military units produced with food; double production speed for castles</English>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIVIC_MERCANTILISM_HELP</Tag>
```

Comment:
Which Mercantilism Help would you like to be displayed. I will remove the other one.